docs/en/setup/service-agent/java-agent/configurations.md

```diff
@@ -128,12 +128,17 @@ This is the properties list supported in `agent/config/agent.config`.
+| `plugin.nettyhttp.filter_length_limit`                           | When `COLLECT_REQUEST_BODY` is enabled, how many characters to keep and send to the OAP backend, use negative values to keep and send the complete body.                                                                                                                                                                                                                                                                                                                                                                                     | SW_PLUGIN_NETTY_HTTP_FILTER_LENGTH_LIMIT                         | `1024`                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+| `plugin.nettyhttp.supported_content_types_prefix`                | When `COLLECT_REQUEST_BODY` is enabled and content-type start with `HTTP_SUPPORTED_CONTENT_TYPES_PREFIX`, collect the body of the request , multiple paths should be separated by `,`                                                                                                                                                                                                                                                                                                                                                          | SW_PLUGIN_NETTY_HTTP_SUPPORTED_CONTENT_TYPES_PREFIX              | `application/json,text/`                                                                                                                                                                                                                                                                                                                                                                                                                             |
```

Comment:
What are changed about this?

Reply:
Sorry, it hasn't changed. Maybe I accidentally formatted it.

Reply:
Please revert them if they are irrelevant.